scantext/templates/scantext/license_scan.html

```diff
@@ -0,0 +1,50 @@
+{% extends "scanpipe/base.html" %} 
+{% load static humanize %} 
+
+{% block content %}
+<div class="container is-max-desktop">
+    {% include 'scanpipe/includes/navbar_header.html' %}
+    <div class="mx-5 mb-2">{% include 'scanpipe/includes/messages.html' %}</div>
+
+    <section class="section pt-0">
+        <div class="is-flex is-justify-content-space-between is-align-items-center mb-2">
+            <div class="mb-2">
+                <h1 class="title is-5">Scan License</h1>
+            </div>
+            <div>
+                <a href="{% url 'project_list' %}" class="button is-link">Go to Projects</a>
```

Comment:
I don't think we need this link anymore since we have a "Projects" link in the header.
Also, we should add a new "Scan" link in the header right next to "Projects" and before "Documentation" that would redirect to the `license_scan` view.
This may change in the future but this will save us some navigation time during the development phase.